src/platform/i386/boot_comp.c

```diff
@@ -116,14 +127,14 @@ kern_boot_comp(void)
         struct captbl *ct;
         unsigned int i;
 	u8_t *boot_comp_captbl;
-	pgtbl_t pgtbl = (pgtbl_t)chal_va2pa(&boot_comp_pgd), untyped_pgd;
 	void *thd_mem, *tcap_mem;
+	pgtbl_t pgtbl   = (pgtbl_t)chal_va2pa(&boot_comp_pgd), boot_vm_pgd;
 	u32_t hw_bitmap = 0xFFFFFFFF;
 
 	printk("Setting up the booter component.\n");
 
 	boot_comp_captbl = mem_boot_alloc(BOOT_CAPTBL_NPAGES);
-        ct = captbl_create(boot_comp_captbl);
+        ct               = captbl_create(boot_comp_captbl);
```

Comment:
yes I noticed that too, I could have indented it by whole but wasn't sure if you'd approve it.